docs/features/coursemanagement/permissions.md

```diff
@@ -2,5 +2,69 @@
 
 # Grant Course Permissions
 
-----------
-TODO add content
\ No newline at end of file
+Platform and course administrators can grant permissions to a user under the course administration menu for a specific course.  
+
+
+![Permission](../../img/course_admin_items/course_permission.png)  
+*Fig. Select the Course Permission from the drop-down list of Course Administration*
+
```

Comment:
This page should be much more abstract. 

I'd suggest sth. like the following:
"The platform offers a very flexible and highly configurable roles and rights management system.
This system comes in a default configuration and can be adjusted according to the needs of the partners separately per platform instance.

Roles can be platform wide or course wide. The permissions that are assigned to each group can be defined and configured very fine-grained. "

You can remove the rest.